src/app/app.module.ts

```diff
@@ -1,20 +1,27 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
 
 import { AppComponent } from './app.component';
 import { NewsComponent } from './components/news/news.component';
-import { HttpClientModule } from '@angular/common/http';
+import { RegionSelectComponent } from './components/region-select/region-select.component';
+
+import { SmartableAiNewsService } from './services/news.service';
+import { RegionSelectService } from './services/region-select.service';
 
 @NgModule({
   declarations: [
     AppComponent,
-    NewsComponent
+    NewsComponent,
+    RegionSelectComponent
   ],
   imports: [
     BrowserModule,
+    FormsModule,
     HttpClientModule
   ],
-  providers: [],
+  providers: [{ provide: 'NewsService', useClass: SmartableAiNewsService }, RegionSelectService],
```

Comment:
I could be wrong, but do we need to specify the providers here?
I was under the impression that just having the ```@Injectable({  providedIn: 'root' })``` decorator is enough for angular to automatically create providers for each service

Reply:
You're right, at least `RegionSelectService` is not needed. Not sure how to handle named injections yet, will have to look it up.